pydantic/deprecated/json.py

```diff
@@ -104,10 +113,16 @@ def custom_pydantic_encoder(type_encoders: Dict[Any, Callable[[Type[Any]], Any]]
         return pydantic_encoder(obj)
 
 
+@deprecated('timedelta_isoformat is deprecated.')
 def timedelta_isoformat(td: datetime.timedelta) -> str:
     """
     ISO 8601 encoding for Python timedelta object.
     """
+    warnings.warn(
+        'timedelta_isoformat is deprecated.',
+        DeprecationWarning,
+        stacklevel=2,
+    )
```

Comment:
```suggestion
    warnings.warn('timedelta_isoformat is deprecated.', DeprecationWarning, stacklevel=2)
```
I think it makes it multiline because of the trailing comma on the final item, if you delete it it will make it one line. @samuelcolvin I think you brought this up before, if you're still open to it I'd be in favor of making it just always reformat to a single line when it can. (That's a separate thing to this PR of course.)